Animal.java

```diff
@@ -1,8 +1,15 @@
 public class Animal {
 
-    public String name;
+    public String nname;
```

Comment:
Can you change this var name back to name, I think it was typo

Reply:
fix